### ISSUE TITLE
Webpack for Mortals

### ISSUE THREAD

**Issue by freestylebit** [dcjs/talks#20]
Your Name: Peter Cho
Your twitter handle: [freestylebit](https://twitter.com/freestylebit)
A few words about yourself: Developer in Alexandria, Expert Gift Card Strategist, Dedicated Traveller to Quality All-You-Can-Eat Buffets, Proud Owner of an Unlimited 4G Data Plan.

Talk title: Webpack for Mortals

Talk abstract:
Curious about how you can get started with Webpack to optimize your front end?  This session serves as a starting point for anyone who is curious about how Webpack can enhance their development workflow and also understand the black magic mysteries behind what it's doing when compiling the source to a bundled distributed collection of files.  Also serves as a good discussion for strategizing how to set up your bundles for an effective front end.

Expected length: 45-60min

Available months: All

Notes: I've given this talk at another meetup, so this could be throw in the backburner as needed, but figured I'd throw this into the mix.

Basically, I follow this repo I prepared to live demo some of the key concepts I've learned: https://github.com/freestylebit/webpack-tutorial

**Comment by jakerella** [comment]
Neat! Thanks Peter. We'll keep this one in mind. We're fleshing out our schedule and will be in touch after the other organizers review.

**Comment by elgreg** [comment]
+1 this sounds like a great subject for our group.

**Comment by samiam151** [comment]
Can't wait for this one.

**Comment by jakerella** [comment]
Hi @freestylebit! Would you be available to deliver this talk for our March meetup? It would be on March 9 at 6:30pm.

**Comment by freestylebit** [comment]
That date should be good for me.  Seems like the webpack world has evolved since I last created the repo, so I'll spend some time this month updating the presentation.  I'll post updates here when I get to it next week.

Thanks!

**Comment by jakerella** [comment]
Fantastic! Consider it set. We'll announce at the February 9 meetup.

**Comment by jakerella** [comment]
This is scheduled! Please be sure to RSVP. https://www.meetup.com/DC-JavaScript/events/237892043/

**Comment by freestylebit** [comment]
I should be RSVPed now.  I'm almost done modifying a couple more things, but I should be ready to go by the end of this week.
